pages/data-cleanup.tsx

```diff
@@ -735,14 +730,45 @@ export default function DataCleanupPage(props) {
                     dropDownItems={dropdownData.writingAdvisors}
                     handleChange={handleChange}
                   ></DropDownField>
+                </div>
 
-                  <TextField
-                    label="Tax ID"
-                    fieldName="tax_id"
-                    fieldValue={formData.contactRecord?.tax_id}
-                    handleChange={handleChange}
-                  ></TextField>
+                <div className={styles.formColumn}>
+                  {isIndividual(formData) ? (
+                    <>
+                      <TextField
+                        label="Tax ID"
+                        fieldName="tax_id"
+                        fieldValue={formData.contactRecord?.tax_id}
+                        handleChange={handleChange}
+                      ></TextField>
+
+                      <DateField
+                        label="Date of Birth"
+                        fieldName="dob"
+                        fieldValue={formData.contactRecord?.dob}
+                        handleDateChange={handleDateChange}
+                      ></DateField>
+
+                      <DropDownField
+                        label="Marital Status"
+                        fieldName="marital_status_id"
+                        fieldValue={formData.contactRecord?.marital_status_id}
+                        dropDownItems={dropdownData.maritalTypes}
+                        handleChange={handleChange}
+                      ></DropDownField>
+                    </>
```

Comment:
This is what I meant, the parathesis here is saying it's a JSX expression which needs a single parent so it's good here